microsoft/testsuites/core/mana.py

```diff
@@ -0,0 +1,164 @@
+import re
+
+from lisa import (
+    LisaException,
+    Node,
+    SkippedException,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.operating_system import Linux, Posix
+from lisa.sut_orchestrator import AZURE
+from lisa.tools import Dmesg, KernelConfig, Modprobe
+
+
+@TestSuiteMetadata(
+    area="mana_driver",
+    category="functional",
+    description="""
+    This test suite checks for the presence of mana and mana_ib drivers
+    """,
+)
+class ManaDriverCheck(TestSuite):
+    MANA_DRIVER_MESSAGE_PATTERN = re.compile(r"\[\s*[0-9.]+\s*\]\s+mana\s+.*")
+    MANA_IB_DRIVER_MESSAGE_PATTERN = re.compile(r"\[\s*[0-9.]+\s*\]\s+mana_ib\s+.*")
+
+    @TestCaseMetadata(
+        description="""
+        This test case checks for the mana ethernet driver.
+        """,
+        priority=4,
```

Comment:
P4 means it's seldom chance to run. If it's not high cost tests like longhaul or stress tests, they don't need to be P4. And for the impact, what if it's failed? P4 means less impact.

Reply:
You're right, it should be alright to run since it skips unless we're on MANA.